android/titanium/src/java/org/appcelerator/titanium/TiC.java

```diff
@@ -1337,11 +1352,21 @@
 	/**
 	 * @module.api
 	 */
+	public static final String PROPERTY_ENABLE_LIGHTS = "enableLights";
+
+	/**
+	 * @module.api
+	 */
 	public static final String PROPERTY_ENABLE_RETURN_KEY = "enableReturnKey";
 
 	/**
 	 * @module.api
 	 */
+	public static final String PROPERTY_ENABLE_VIBRATION = "enableVibrations";
```

Comment:
The property string is set to "enableVibrations" with an 's', but you document this property without an 's'. This is a typo, right?